test/jenkins/criu-iter.sh

```diff
@@ -5,4 +5,9 @@ set -e
 source `dirname $0`/criu-lib.sh
 prep
 mount_tmpfs_to_dump
-./test/zdtm.py run --all --keep-going --report report --parallel 4 --iter 3 -x 'maps04' || fail
+
+ITER_EXCLUDE="-x maps02"
+# Exclude those tests from multiple iterations because their hooks are
+# dependant on the exact order of the different stages
+ITER_EXCLUDE="$ITER_EXCLUDE -x net_lock_socket_iptables -x net_lock_socket_nftables"
+./test/zdtm.py run --all --keep-going --report report --parallel 4 --iter 3 $ITER_EXCLUDE || fail
```

Comment:
Hmm, the tests are passing when I run them with multiple iterations locally.

```console
sudo ./zdtm.py run -T 'zdtm/static/net_lock_socket_*' --parallel 4 --iter 3

Checking feature network_lock_nftables
network_lock_nftables is supported
userns is supported
=== Run 1/2 ========-------- zdtm/static/net_lock_socket_nftables
timens isn't supported on 5.13.4-200.fc34.x86_64
================ Run zdtm/static/net_lock_socket_nftables in h =================
Start test
Test is SUID
./net_lock_socket_nftables --pidfile=net_lock_socket_nftables.pid --outfile=net_lock_socket_nftables.out
Running zdtm/static/net_lock_socket_nftables.hook(--post-start)
Running zdtm/static/net_lock_socket_nftables.hook(--pre-dump)
Run criu dump
Running zdtm/static/net_lock_socket_nftables.hook(--pre-restore)
Run criu restore
Running zdtm/static/net_lock_socket_nftables.hook(--post-restore)
Running zdtm/static/net_lock_socket_nftables.hook(--pre-dump)
Run criu dump
Running zdtm/static/net_lock_socket_nftables.hook(--pre-restore)
Run criu restore
Running zdtm/static/net_lock_socket_nftables.hook(--post-restore)
Running zdtm/static/net_lock_socket_nftables.hook(--pre-dump)
Run criu dump
Running zdtm/static/net_lock_socket_nftables.hook(--pre-restore)
Run criu restore
Running zdtm/static/net_lock_socket_nftables.hook(--post-restore)
Send the 15 signal to  57
Wait for zdtm/static/net_lock_socket_nftables(57) to die for 0.100000
Running zdtm/static/net_lock_socket_nftables.hook(--clean)
Removing dump/zdtm/static/net_lock_socket_nftables/57
================ Test zdtm/static/net_lock_socket_nftables PASS ================

userns is supported
=== Run 2/2 ================ zdtm/static/net_lock_socket_iptables
timens isn't supported on 5.13.4-200.fc34.x86_64
================ Run zdtm/static/net_lock_socket_iptables in h =================
Start test
Test is SUID
./net_lock_socket_iptables --pidfile=net_lock_socket_iptables.pid --outfile=net_lock_socket_iptables.out
Running zdtm/static/net_lock_socket_iptables.hook(--post-start)
Running zdtm/static/net_lock_socket_iptables.hook(--pre-dump)
Run criu dump
Running zdtm/static/net_lock_socket_iptables.hook(--pre-restore)
Run criu restore
Running zdtm/static/net_lock_socket_iptables.hook(--post-restore)
Running zdtm/static/net_lock_socket_iptables.hook(--pre-dump)
Run criu dump
Running zdtm/static/net_lock_socket_iptables.hook(--pre-restore)
Run criu restore
Running zdtm/static/net_lock_socket_iptables.hook(--post-restore)
Running zdtm/static/net_lock_socket_iptables.hook(--pre-dump)
Run criu dump
Running zdtm/static/net_lock_socket_iptables.hook(--pre-restore)
Run criu restore
Running zdtm/static/net_lock_socket_iptables.hook(--post-restore)
Send the 15 signal to  57
Wait for zdtm/static/net_lock_socket_iptables(57) to die for 0.100000
Running zdtm/static/net_lock_socket_iptables.hook(--clean)
Removing dump/zdtm/static/net_lock_socket_iptables/57
================ Test zdtm/static/net_lock_socket_iptables PASS ================
```

Reply:
I am sorry, this is my bad. I assumed that it would fail the same as the netns tests did.